examples/arc20/README.md

```diff
@@ -0,0 +1,13 @@
+# arc20.aleo
+
+## Build Guide
+
+To compile this Aleo program, run:
+```bash
+snarkvm build
+```
+
+To execute this Aleo program, run:
+```bash
+snarkvm run hello
```

Comment:
@ghostant-1017 can you add a few more commands to the readme on running the different functions in this program?

Reply:
Okay, I'll add it later